modules/ROOT/pages/platform/metrics-integration/introduction.adoc

```diff
@@ -0,0 +1,194 @@
+
+[NOTE]
+====
+The revocation described takes effect after the authorization caches expire, which takes approximately 5 minutes.
+It results in HTTP 401 being returned, along with the message `User doesn't have access to Metrics resources.`
+However, if you remove only the Aura API credentials used to retrieve metrics, the revocation will take effect only after the tokens issued with these credentials expire, as no new token can be issued anymore. Currently used token expiration time is 1 hour.
+====
+
+== Metric details
+
+[aura-cmi-metric-labels]
+.Metric labels
+
+Depending on the metric, the following labels are applied:
+
+* `aggregation`: the aggregation used to calculate the metric value, set on every metric.
+Since the Neo4j instance is deployed as a Neo4j cluster, aggregations are performed to combine values from all relevant cluster nodes.
+The following aggregations are used: `MIN`, `MAX`, `AVG` and `SUM`.
+* `instance_id`: the Aura instance ID the metric is reported for, set on every metric.
+* `database`: the name of the Neo4j database the metric is reported for.
+Set to `neo4j` by default.
+
+.Example
+
+[source, shell]
+----
+# HELP neo4j_database_count_node The total number of nodes in the database.
+# TYPE neo4j_database_count_node gauge
+neo4j_database_count_node{aggregation="MAX",database="neo4j",instance_id="78e7c3e0"} 778114.000000 1711462853000
+----
+
+[aura-cmi-looking-up-metric-name]
+.Looking up metric name in Neo4j Aura Advanced Metrics
+
+In Aura Console classic experience Advanced Metrics, it is possible to find out the metric name that corresponds to the chart, by using the chart menu item "Metrics Integration" as shown.
+
+image::advanced_metrics.png[]
+
+Or the Metrics reference provided xref:./reference.adoc[here].
+
+[aura-cmi-metric-scrape-interval]
+== Metric scrape interval
+
+Recommended scrape interval for metrics is in the range of 30 seconds up to 2 minutes, depending on requirements.
+The metrics endpoint caches metrics for 30 seconds.
+
+[aura-cmi-metrics-granularity]
+== Metric granularity
+
+The metrics returned by the integration endpoint provide the labels: `aggregation`, `instance_id`, and `database`.
+
+An Aura instance typically runs on multiple servers to achieve availability and workload scalability in different modes.
+These servers are deployed across different Cloud Provider availability zones in the user-selected region.
+
+Metrics Integration supports a more granular view of the Aura instance metrics with additional data points and labels for availability zone & instance mode combinations.
+This view can be enabled on demand.
+
+[NOTE]
+====
+Contact link:https://support.neo4j.com/[Customer Support] to enable more granular metrics of instances for your project.
+====
+
+[NOTE]
+====
+There may be a delay in more granular metrics being available when a new Aura instance is created.
+This is because of the way 'availability zone' data is collected.
+====
```

Comment:
```suggestion

Note that there may be a delay in more granular metrics being available when a new Aura instance is created.
This is because of the way 'availability zone' data is collected.
```

Multiple admonitions are not advisable from a UX standpoint.